### PR TITLE
Adding mariner in dapr init docs

### LINKED ISSUE #2747: Mariner support in CLI
**What content needs to be created or modified?**

Mariner support in CLI, `--image-variant` flag and usage.
**Describe the solution you'd like**

Add docs for the new flag.
**Where should the new material be placed?**

CLI references section.
**The associated pull request from dapr/dapr, dapr/components-contrib, or other Dapr code repos**
https://github.com/dapr/cli/pull/1029

### PR DESCRIPTION
**Pull request by shivamkm07**
Signed-off-by: shivam <shivamkm07@gmail.com>

Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [ ] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [ ] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

Adding `--image-variant` flag in dapr init docs

## Issue reference

Please reference the issue this PR will close: #2747

### CODE REVIEW

File: daprdocs/content/en/reference/cli/dapr-init.md [hunk 1]
```diff
@@ -59,6 +60,12 @@ You can also specify a specific runtime version. Be default, the latest version
 dapr init --runtime-version 1.4.0
 ```
 
+You can also install Dapr with a particular image variant, for example: `mariner`.
```
Comment (msfussell) [inline daprdocs/content/en/reference/cli/dapr-init.md]: can you make Mariner a reference link here to these docs https://docs.dapr.io/operations/hosting/kubernetes/kubernetes-deploy/#using-mariner-based-images

Comment (msfussell) [reply daprdocs/content/en/reference/cli/dapr-init.md]: @shivamkm07 - This change is for the v1.9 release, yes? If so, can you target this docs for v1.9 branch?

**Comment by greenie-msft** [comment]
@shivamkm07 please see @msfussell's comment

**Comment by msfussell** [comment]
@shivamkm07 - Ping on this docs issue?

**Comment by hhunter-ms** [comment]
@mukundansundar @shivamkm07 Just want to check if this should be targeting v1.9, as it's marked under the 1.9 milestone?

**Comment by mukundansundar** [comment]
@hhunter-ms yes this is targeting 1.9

**Comment by msfussell** [comment]
@shivamkm07 - We need to release the docs. Please point this PR at the v1.9 branch.

File: daprdocs/content/en/reference/cli/dapr-init.md (hunk 1 above)
Comment (shivamkm07) [reply daprdocs/content/en/reference/cli/dapr-init.md]: Mariner reference added, Thanks!

**Comment by shivamkm07** [comment]
@msfussell @hhunter-ms Rebased PR to v1.9 and included the comments. Thanks!

**Comment by msfussell** [comment]
@hhunter-ms  @shivamkm07 Did not see this added per my comment above ^^
"Can you make Mariner a reference link here to these docs https://docs.dapr.io/operations/hosting/kubernetes/kubernetes-deploy/#using-mariner-based-images"

File: daprdocs/content/en/reference/cli/dapr-init.md [hunk 2]
```diff
@@ -59,6 +60,12 @@ You can also specify a specific runtime version. Be default, the latest version
 dapr init --runtime-version 1.4.0
 ```
 
+You can also install Dapr with a particular image variant, for example: [mariner](https://docs.dapr.io/operations/hosting/kubernetes/kubernetes-deploy/#using-mariner-based-images).
```
Comment (greenie-msft) [inline daprdocs/content/en/reference/cli/dapr-init.md]:
```suggestion
You can also install Dapr with a particular image variant, for example: [mariner]({{< ref "kubernetes-deploy.md#using-mariner-based-images" >}}).
```

**Comment by shivamkm07** [comment]
> @hhunter-ms @shivamkm07 Did not see this added per my comment above ^^ "Can you make Mariner a reference link here to these docs https://docs.dapr.io/operations/hosting/kubernetes/kubernetes-deploy/#using-mariner-based-images"

Done. Thanks @greenie-msft

**Review by greenie-msft — APPROVED** [review]

**Pull request merged**